Fix compile time mismatched types error when running "cargo build --all --all-targets  --release"

When running the command `cargo build --all --all-targets  --release`, I see this error below. It is likely from the recent ACL change and should be a small change to fix the casting.



```
error[E0308]: mismatched types
   --> src/context/mod.rs:504:74
    |
504 |             let command = raw::RedisModule_GetCommand.unwrap()(self.ctx, command_name);
    |                           ------------------------------------           ^^^^^^^^^^^^ expected `*const u8`, found `*const i8`
    |                           |
    |                           arguments to this function are incorrect
    |
    = note: expected raw pointer `*const u8`
               found raw pointer `*const i8`

error[E0308]: mismatched types
   --> src/context/mod.rs:505:72
    |
505 |             raw::RedisModule_SetCommandACLCategories.unwrap()(command, acl_flags).into()
    |             -------------------------------------------------          ^^^^^^^^^ expected `*const u8`, found `*const i8`
    |             |
    |             arguments to this function are incorrect
    |
    = note: expected raw pointer `*const u8`
               found raw pointer `*const i8`
```

--- a/src/context/mod.rs
+++ b/src/context/mod.rs
@@ -499,7 +499,7 @@ impl Context {
         feature = "min-redis-compatibility-version-7-2",
         feature = "min-valkey-compatibility-version-8-0"
     ),))]
-    pub fn set_acl_category(&self, command_name: *const i8, acl_flags: *const i8) -> raw::Status {
+    pub fn set_acl_category(&self, command_name: *const c_char, acl_flags: *const c_char) -> raw::Status {
         unsafe {
             let command = raw::RedisModule_GetCommand.unwrap()(self.ctx, command_name);
             raw::RedisModule_SetCommandACLCategories.unwrap()(command, acl_flags).into()

